src/main/java/seedu/address/logic/commands/edit/EditModuleRoleOperation.java

```diff
@@ -33,4 +37,41 @@ public abstract class EditModuleRoleOperation {
     public static boolean isValidModuleRoleOperation(String operation) {
         return operation.toLowerCase().matches(VALIDATION_REGEX);
     }
+
+    /**
+     * Returns a description of the change in module roles.
+     * @param moduleRoleMapBefore The module role map before the change.
+     * @param moduleRoleMapAfter The module role map after the change.
+     * @return A description of the change in module roles.
+     */
+    public static String getModuleCodeChangeDescription(
+            ModuleRoleMap moduleRoleMapBefore, ModuleRoleMap moduleRoleMapAfter) {
+
+        StringBuilder stringBuilderAdded = new StringBuilder();
+        boolean hasAdded = false;
+        for (ModuleRolePair moduleRolePair : moduleRoleMapAfter.getData()) {
+            if (!moduleRoleMapBefore.containsModuleRolePair(moduleRolePair)) {
+                hasAdded = true;
+                stringBuilderAdded.append(moduleRolePair).append(" ");
+            }
+        }
+
+        StringBuilder stringBuilderDeleted = new StringBuilder();
+        boolean hasDeleted = false;
+        for (ModuleRolePair moduleRolePair : moduleRoleMapBefore.getData()) {
+            if (!moduleRoleMapAfter.containsModuleRolePair(moduleRolePair)) {
+                hasDeleted = true;
+                stringBuilderDeleted.append(moduleRolePair).append(" ");
+            }
+        }
+        List<String> finalDescription = new ArrayList<>();
+        if (hasAdded) {
+            finalDescription.add("Module role added: " + stringBuilderAdded.toString().strip());
+        }
+        if (hasDeleted) {
+            finalDescription.add("Module role deleted: " + stringBuilderDeleted.toString().strip());
```

Comment:
Would it better to changes the messages to `Module role(s) added` and `Module role(s) deleted` instead?